apps/pom.xml

```diff
@@ -88,22 +88,22 @@
 		<sonar.jacoco.reportPath>target/jacoco.exec</sonar.jacoco.reportPath>
 		<sonar.jacoco.itReportPath>target/jacoco-it.exec</sonar.jacoco.itReportPath>
 
-		<metrics.version>4.1.29</metrics.version>
-		<newrelic.version>0.5.0</newrelic.version>
+		<metrics.version>4.2.1</metrics.version>
+		<newrelic.version>0.7.0</newrelic.version>
 		<dropwizard-metrics-newrelic.version>0.7.0</dropwizard-metrics-newrelic.version>
 		<telemetry-http-okhttp.version>0.15.0</telemetry-http-okhttp.version>
-		<aws-java-sdk-bom.version>1.12.245</aws-java-sdk-bom.version>
+		<aws-java-sdk-bom.version>1.12.276</aws-java-sdk-bom.version>
 		<spring-data-jpa.version>2.7.0</spring-data-jpa.version>
-		<hibernate.version>5.6.1.Final</hibernate.version>
+		<hibernate.version>5.6.10.Final</hibernate.version>
 		<junit.version>5.8.2</junit.version>
 		<jackson.version>2.13.3</jackson.version>
 		<mockito.version>4.2.0</mockito.version>
-		<postgresql.version>42.2.20</postgresql.version>
+		<postgresql.version>42.4.1</postgresql.version>
 		<hsqldb.version>2.6.0</hsqldb.version>
-		<slf4j.version>1.7.7</slf4j.version>
-		<hikari.version>4.0.3</hikari.version>
+		<slf4j.version>1.7.36</slf4j.version>
+		<hikari.version>5.0.1</hikari.version>
 		<flyway.version>8.5.13</flyway.version>
-		<newrelicAgent.version>7.5.0</newrelicAgent.version>
+		<newrelicAgent.version>7.9.0</newrelicAgent.version>
```

Comment:
should be renamed to newrelic.agent.version

Reply:
Updated